packages/@azure/servicebus/data-plane/examples/javascript/gettingStarted/sendWithInteractiveAuth.js

```diff
@@ -0,0 +1,56 @@
+/*
+  This sample demonstrates how to create a namespace using TokenCredentials.
+
+  Setup: AAD RBAC is enabled only on the new namespaces in these regions
+    - [US East, US East 2, or West Europe] for the preview.
+    Please ensure that your servicebus-namespace is present in one of these regions.
+*/
+
+const { Namespace } = require("@azure/service-bus");
+const { interactiveLogin } = require("ms-rest-azure");
+
+const aadServiceBusAudience = "https://servicebus.azure.net/";
+// Define Service Bus Endpoint here and related entity names here
+const serviceBusEndpoint = ""; // <your-servicebus-namespace>.servicebus.windows.net
+const queueName = "";
+
+const listOfScientists = [
+  { name: "Einstein", firstName: "Albert" },
+  { name: "Heisenberg", firstName: "Werner" },
+  { name: "Curie", firstName: "Marie" }
+];
+
+async function main(){
+  // tokenCreds can be of the following types
+  //      ApplicationTokenCredentials
+  //      UserTokenCredentials
+  //      DeviceTokenCredentials
+  //      MSITokenCredentials
+  // interactiveLogin returns DeviceTokenCredentials
+  const tokenCreds = await interactiveLogin({
+    tokenAudience: aadServiceBusAudience
+  });
+
+  const ns = Namespace.createFromAadTokenCredentials(serviceBusEndpoint, tokenCreds);
+  const client = ns.createQueueClient(queueName);
```

Comment:
I wonder if we should even bother with queueClients and senders here.

The main intention of this sample is authentication.

How about the below?

```
const ns = Namespace.createFromAadTokenCredentials(serviceBusEndpoint, tokenCreds);

// Refer to other samples, and place your code here to create queue clients, and then send/receive messages

await ns.close();
```

Reply:
And then rename the file to simply `interactiveLogin.js`

Reply:
Same for the other file